use-cases/personalization-sending-two-emails-to-two-groups-recipients-from-two-different-from-emails.md

```diff
@@ -0,0 +1,73 @@
+
+### Personalization (with helper) - Sending Two Different Emails to Two Different Groups of Recipients from two different From emails
```

Comment:
These are lengthy examples for relatively small new functionality. Prefer to keep these pretty short to make it move obvious what the effective lines are.